### PR TITLE
Diagram Database

### PR DESCRIPTION
**Pull request by Arch-Noize**
We produced a database similar to that of a given [diagram](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/images/clinic_diagram.png)

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ADD CONSTRAINT treatment_id_fk FOREIGN KEY(treatment_id)
+REFERENCES treatments(id);
+
+CREATE INDEX index_treatment ON invoice_items(treatment_id);
+
+CREATE INDEX index_medical_history ON invoices(medical_history_id);
+
+CREATE INDEX index_invoice ON invoice_items(treatment_id);
+
+CREATE INDEX index_patient ON medical_histories(patient_id);
```
Comment (V-Blaze) [inline schema_based_on_diagram.sql]: - You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] You did great creating indexes for the `FOREIGN KEYS` in each table , but you did create index for the Keys in the `med_history_treatment` joint table. I'll suggest you add the create the indes for the two ids. :+1: :waffle:

File: README.md [hunk 2]
```diff
@@ -286,6 +159,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (V-Blaze) [inline README.md:161]: - [ ] The MIT link in your Readme file leads to a 404 Error page, I'll suggest you add a license file to your project and link it properly as broken links in the readme make your repository look a bit unprofessional. :smile: :small_airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Arch-Noize, @lilskyex0x  

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- All table created correctly ✔
-  "join table" for the many-to-many relationship is created👋 
-  Correct git flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @Arch-Noize 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- [Optional] Please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**